### ISSUE TITLE
Scale factor on centr_clo

### ISSUE THREAD

**Issue by raffam** [igraph/rigraph#334]
I think `centr_clo` needs a `scale` parameter like `centr_eigen` has. In case the parameter is set to true, the closeness centrality of the single nodes are not scaled (that is, not multiplied by N-1, where N is the number of nodes)

The `normalized` parameter which controls whether the _graph_ centrality score is scaled would stay.

**Comment by harryprince** [comment]
@raffam you can push a Pull Request, :)

**Comment by szhorvat** [comment]
What is being asked here is that `centr_clo()` should have the equivalent of the `normalized` parameter of `closeness()`. Naming this `scale` is not ideal, as the parameter name would differ between `centr_clo()` and `closeness()`. However, `centr_clo()` already has a parameter named `normalized`, which performs a different function.

My personal opinion on this is that there is little value in implementing this since:

 - What is being asked for is easily accomplished using the `closeness()` and `centralize()` functions.
 - Centralization is a niche concept. IMO its formulation is rather arbitrary.

I suggest closing.

**Comment by schochastics** [comment]
I agree with @szhorvat on this and close the issue. Reopen if needed
